build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
   compile 'org.jdbi:jdbi3-postgres:3.5.1'
   compile 'org.jdbi:jdbi3-sqlobject:3.5.1'
   compile 'org.postgresql:postgresql:42.2.5'
+  compile 'org.projectlombok:lombok:1.18.4'
```

Comment:
`lombok` is only a compile time dependency :) (see [gradle setup](https://projectlombok.org/setup/gradle))

```
compileOnly 'org.projectlombok:lombok:1.18.4'
```

Reply:
nice, done :)